db/migrate/20150714234345_devise_invitable_add_to_users.rb

```diff
@@ -0,0 +1,27 @@
+class DeviseInvitableAddToUsers < ActiveRecord::Migration
+  def up
+    change_table :users do |t|
+      t.string     :invitation_token
+      t.datetime   :invitation_created_at
+      t.datetime   :invitation_sent_at
+      t.datetime   :invitation_accepted_at
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.